doc/project/dependencies.md

```diff
@@ -30,6 +30,7 @@ Use [scalajs-bundler](https://scalacenter.github.io/scalajs-bundler/) instead.
 Instead, it is provided by [sbt-jsdependencies](https://github.com/scala-js/jsdependencies).
 
 The rest of this section applies to Scala.js 0.6.x only.
+[Scala.js 0.6.x has reached End of Life.]({{ BASE_PATH }}/doc/internals/scalajs-0.6.x-eol.html)
```

Comment:
Instead of this, I propose we completely remove the "Deprecated: Depending on JavaScript libraries" section (and maybe add a basic intro to `scalajs-bundler`)?

I now realize that this is probably better done in another PR.

Reply:
Yes, more generally, I'd like to only show docs for 1.x by default, and have a checkbox to show information related to 0.6.x. A lightweight variant of versioned docs, in some sense.